guides/micronaut-data-jdbc-repository/micronaut-data-jdbc-repository.adoc

```diff
@@ -125,70 +141,50 @@ callout:http-request[3]
 <4> If you care just about the object in the response, use `retrieve`.
 <5> Sometimes, receiving just the object is not enough, and you need information about the response. In this case, instead of `retrieve`, use the `exchange` method.
 
-include::{commondir}/common-testApp.adoc[]
-
-include::{commondir}/common-runapp.adoc[]
+To configure the MySQL Testcontainer, create `application-test.yml` in the test's resources folder.
```

Comment:
test's -> test

Reply:
actually, nvm - delete this whole part. As above, the generated app has a test config file `application-test.yml` with the same content as what's displayed. Either delete this, or keep and rephrase to highlight anything interesting if needed